### PR TITLE
BETA: Register kryptorian.is-a.dev

### PR DESCRIPTION
**Pull request by kryptorian**
Added `kryptorian.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/kryptorian.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "kryptorian",
+        "email": "kryptoria@protonmail.com"
+    },
+    "record": {
+        "A": ["130.61.159.247"]
```
Comment (Stef-00012) [inline domains/kryptorian.json:7]: I cannot reach your website
> 130.61.159.247 didn’t send any data.

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/kryptorian.json (hunk 1 above)
Comment (kryptorian) [reply domains/kryptorian.json:7]: http://130.61.159.247:3000/
website is listening on port 3000

**Review by kryptorian — COMMENTED** [review]
added port

File: domains/kryptorian.json [hunk 2]
```diff
@@ -4,6 +4,6 @@
         "email": "kryptoria@protonmail.com"
     },
     "record": {
-        "A": ["130.61.159.247"]
+        "A": ["130.61.159.247:3000"]
```
Comment (CuteDog5695) [inline domains/kryptorian.json:7]: yea you cant add ports to A record

File: domains/kryptorian.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "kryptorian",
+        "email": "kryptoria@protonmail.com"
+    },
+    "record": {
+        "A": ["130.61.159.247:3000"]
```
Comment (Stef-00012) [inline domains/kryptorian.json:7]: You can't put the port in an A record

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by Stef-00012** [comment]
> added port

it was enough to just tell it in a comment, you didn't need to add it to the record

**Comment by kryptorian** [comment]
> > added port
> 
> it was enough to just tell it in a comment, you didn't need to add it to the record

sry didnt know that was enough :) If i need to do anything please tell me

File: domains/kryptorian.json (hunk 3 above)
Comment (kryptorian) [reply domains/kryptorian.json]: removed

**Review by Stef-DP — APPROVED** [review]
seems good now

**Review by iostpa — APPROVED** [review]
since website is hosted on the IP but on a port, it should be fine

**Pull request merged**